dev_docs/tutorials/adding_performance_metrics.mdx

```diff
@@ -0,0 +1,68 @@
+---
+id: kibDevTutorialAddingPerformanceMetrics
+slug: /kibana-dev-docs/tutorial/adding_performance_metrics
+title: Adding Performance Metrics
+summary: Learn how to instrument your code and analyze performance
+date: 2022-07-07
+tags: ['kibana', 'onboarding', 'setup', 'performance', 'development', 'telemetry']
+---
+
+Reporting performance events are easy as it can be seen below, however it's highly suggested to follow guidelines for what to report in certain fields:
+
+```typescript
+import { reportPerformanceMetricEvent } from '@kbn/ebt-tools';
+
+reportPerformanceMetricEvent(analytics, {
+  eventName: KIBANA_LOADED_EVENT,
+  meta: {
```

Comment:
I'd add a clarification that anything that goes into `meta` is not aggregateable, only searchable.

```suggestion
  meta: {
    // fields that are searchable-as-keywords but not aggregateable
```